doc/man3/PKCS12_newpass.pod

```diff
@@ -17,6 +17,16 @@ PKCS12_newpass() changes the password of a PKCS12 structure.
 B<p12> is a pointer to a PKCS12 structure. B<oldpass> is the existing password
 and B<newpass> is the new password.
 
+=head1 NOTES
+
+Both B<oldpass> and B<newpass> are NUL-terminated strings in the UTF-8 encoding.
+If either is not a valid UTF-8 byte sequence, then that one is interpreted
+as ISO8859-1 instead.
+
+In particular, this means that passwords obtained through the default
+UI_OpenSSL() method, which are in the locale character set, should not be
+used without potentially being converted to UTF-8 first.
```

Comment:
A ISO 8859-1 string would have to be *very carefully* crafted to be taken for a UTF-8 string...